includes/ApplHandlingByID.php

```diff
@@ -299,6 +299,163 @@ function accept ( $id, $type, $user )
 }
 
 
+function forwardApprover ( $id, $type, $user )
+{
+	require "../../LocalSettings.php";
+	require "../Globals.php";
+
+	$sqlconn = new mysqli ( $eorgDBserver, $eorgDBuser, $eorgDBpasswd, $eorgDBname );
+
+	if ( $sqlconn->connect_errno )
+	{
+		echo "Internal Server Error, Sorry for inconvenience.";
+		die();
+	}
+
+	$qry = "SELECT $stat, $AppDate FROM $type WHERE $AppId = ?";
+	$stmt = $sqlconn->prepare ( $qry );
+
+	$stmt->bind_param ( 's', $id );
+	if ( ! $stmt->execute() )
+	{
+		echo "Sorry, there is a problem with database.";
+		die();
```

Comment:
__[phpmd]__
* The function forwardApprover() contains an exit expression.

<!-- Comment by [SideCI](https://sideci.com) -->